### PR TITLE
kernel-uki: drop dbus in initrd

### PR DESCRIPTION
**Pull request by jozzsi**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./LICENSES-AND-NOTICES/SPECS/data/licenses.json`, `./LICENSES-AND-NOTICES/SPECS/LICENSES-MAP.md`, `./LICENSES-AND-NOTICES/SPECS/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

dbus is known to have limitations and usually only gets included for either networking or bluetooth support. 
Please see https://github.com/dracutdevs/dracut/issues/2378 and https://github.com/dracut-ng/dracut-ng/issues/60 .

###### Does this affect the toolchain?  
**YES**

###### Test Methodology
@ddstreet  @trungams - would you please be able to help testing this change given the concerns ?  If dbus indeed can be dropped, than it should be dropped and I can finish up the PR with the rest of the changes.

### CODE REVIEW

**Comment by trungams** [comment]
@jozzsi Change looks good to me. No issue in our initrd

File: toolkit/resources/manifests/package/toolchain_x86_64.txt [hunk 1]
```diff
@@ -162,7 +162,7 @@ itstool-2.0.7-1.azl3.noarch.rpm
 kbd-2.2.0-2.azl3.x86_64.rpm
 kbd-debuginfo-2.2.0-2.azl3.x86_64.rpm
 kernel-cross-headers-6.6.47.1-2.azl3.noarch.rpm
```
Comment (trungams) [inline toolkit/resources/manifests/package/toolchain_x86_64.txt:164]:
```suggestion
kernel-cross-headers-6.6.47.1-3.azl3.noarch.rpm
```

**Review by trungams — CHANGES_REQUESTED** [review]

**Comment by trungams** [comment]
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=638284&view=results

**Review by trungams — APPROVED** [review]

**Review by Camelron — APPROVED** [review]

File: SPECS-SIGNED/kernel-signed/kernel-signed.spec [hunk 2]
```diff
@@ -145,6 +145,9 @@ echo "initrd of kernel %{uname_r} removed" >&2
 %exclude /module_info.ld
 
 %changelog
+* Thu Sep 10 2024 Jo Zzsi <jozzsicsataban@gmail.com> - 6.6.47.1-3
```
[the remark this answers is not in the document]
Comment (jozzsi) [reply SPECS-SIGNED/kernel-signed/kernel-signed.spec]: Thanks. Fixed to Tue.

**Comment by trungams** [comment]
@jozzsi there seems to be a conflict as another PR that went in also bumped the kernel version. Could you bump the release version to 4?

**Comment by jozzsi** [comment]
> @jozzsi there seems to be a conflict as another PR that went in also bumped the kernel version. Could you bump the release version to ~4~ 5?

Sorry, but this process is too much overhead for me as now it also needs a rebase. Feel free to close this PR or take it over.

**Comment by trungams** [comment]
No problem. I can take over the PR. Thank you for the change!

**Pull request merged**